vllm/entrypoints/openai/protocol.py

```diff
@@ -414,6 +415,14 @@ class ChatCompletionRequest(OpenAIBaseModel):
         default=None,
         description="KVTransfer parameters used for disaggregated serving.")
 
+    vllm_xargs: Optional[dict[str, Union[str, int, float]]] = Field(
+        default=None,
+        description=(
+            "Dict of arbitrary additional keyword args with string or numeric "
+            "values. Can be used by custom sampling implementations, plugins, "
+            "etc. Not used by any in-tree sampling implementations."),
```

Comment:
Suggest simplification
```suggestion
            "Additional request parameters with string or "
            "numeric values, used by custom extensions."),
```